lib/cli.js

```diff
@@ -83,6 +83,12 @@ function cli()
 				info: "URL to an HTML document.",
 				type: require("url")
 			},
+			"maxSocketsPerHost":
```

Comment:
This isn't very CLI-like and `--max-sockets-per-host` is too long.

Reply:
what about `--concurrent-requests` or `--sockets-per-host`?

Reply:
Not happy with those either, heh. This is part of the reason why I didn't get around to adding this option to the CLI.

Reply:
what do you think about adding a options which allows to load a json file like defaultOption which can then define any option available?

Reply:
I'd have to think about that one. It might be nice long term, but I think that most people, including myself, would prefer not to use files to access a CLI.

Reply:
I don't like `max-sockets-per-host`.

Reply:
`--socket-limit` ? `--per-host-limit`?

Reply:
Maybe `--host-requests`.